explore/circadian-efficacy/data/tissue_uberon.tsv

```diff
@@ -0,0 +1,14 @@
+Tissue	Exact.Uberon.code	Exact.Uberon.term	Hetionet.Uberon.code	Hetionet.Uberon.term	GTEx.name

```

Comment:
in general, we should use all lowercase underscore separated column names like `exact_uberon_term`. The exception is if we have inherited a column name from an input resource.

should `Tissue` be `circadb_tissue` for clarity?